### ISSUE TITLE
Question: Multiplatform?

### ISSUE THREAD

**Issue by DioManquicio** [binogure-studio/city-game-studio-i18n#468]
One question, I saw that in a question I mentioned the developer, about the multiplatform for the development of video games from a certain time, but I have the doubt if it is definitive that it was implemented in the game or it was only a possibility.

Translated with the google translator, sorry for the bad english

**Comment by xsellier** [comment]
Hello,

When you make a game you make it for a single platform. Then, you can put the game on hold and start porting it onto several platforms. And once everything is ready to be released, you can release the game for all the platforms thanks to a single button.

However, if you want to release your game as soon as it is ready, you can also do it.

**Comment by DioManquicio** [comment]
well ok
